interface/forms/requisition/view.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Need a license/author/copyright header here
(can just use same one you have in the new.php script)